Remotion/Data/DomainObjects.UnitTests/Infrastructure/DomainObjectTransactionContextTest.cs

```diff
@@ -288,6 +290,83 @@ private void DeleteOrder (Order order)
       order.Delete();
     }
 
+    [Test]
+    public void AllowedMembers ()
```

Comment:
We need to slighlty tweak the tests that were moved over from the deleted `Initialzing`-state version of the Context.
```suggestion
    public void AllowedMembers_WhileReferenceInitializingIsActive ()
```